BugReporter/UserEnvironmentInformation.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Text;
+using GitCommands;
+using GitExtUtils;
+using GitExtUtils.GitUI;
+
+namespace BugReporter
+{
+    internal static class UserEnvironmentInformation
```

Comment:
Do we still need the `GitUI\UserEnvironmentInformation.cs`?

Reply:
Yes, because it populates information for other purposes too (e.g. the About dialog).
I wanted to reuse it across the projects, but it didn't quite work.